executor/cmd/executor/main.go

```diff
@@ -71,8 +71,8 @@ var (
 	filename          = flag.String("file", "", "Load graph from file")
 	hostname          = flag.String("hostname", "", "The hostname of the running server")
 	logWorkers        = flag.Int("logger_workers", 10, "Number of workers handling payload logging")
-	logWorkBufferSize = flag.Int("log_work_buffer_size", 10000, "Limit of buffered logs in memory while waiting for downstream request ingestion")
-	logWriteTimeoutMs = flag.Int("log_write_timeout_ms", 2000, "Timeout before giving up writing log if buffer is full. If <= 0 will immediately drop log on full log buffer.")
+	logWorkBufferSize = flag.Int("log_work_buffer_size", loghandler.DefaultWorkQueueSize, "Limit of buffered logs in memory while waiting for downstream request ingestion")
```

Comment:
Do these have to be params? Wondering whether env vars could be enough

Reply:
As we have annotations at present do we need to add this as well?